darwin/dataset/remote_dataset_v2.py

```diff
@@ -135,11 +140,19 @@ def get_releases(self) -> List["Release"]:
             Release.parse_json(self.slug, self.team, payload)
             for payload in releases_json
         ]
-        return sorted(
-            filter(lambda x: x.available, releases),
-            key=lambda x: x.version,
-            reverse=True,
-        )
+
+        if retry:
+            return sorted(
+                releases,
+                key=lambda x: x.version,
+                reverse=True,
+            )
+        else:
+            return sorted(
+                filter(lambda x: x.available, releases),
```

Comment:
I'm sure you can have this nicer where only the first argument `releases` or `filter(lambda x: x.available, releases)` is chosen with an `if` and have a single return line:

```
return sorted(
                releases_fn,
                key=lambda x: x.version,
                reverse=True,
            )
```